packages/web-components/fast-foundation/docs/api-report.md

```diff
@@ -634,6 +718,54 @@ export const RadioGroupTemplate: import("@microsoft/fast-element").ViewTemplate<
 // @public
 export const RadioTemplate: import("@microsoft/fast-element").ViewTemplate<Radio, any>;
 
+// Warning: (ae-different-release-tags) This symbol has another declaration with a different release tag
+// Warning: (ae-internal-mixed-release-tag) Mixed release tags are not allowed for "Select" because one of its declarations is marked as @internal
+//
+// @public
+export class Select extends Listbox {
+    // @internal (undocumented)
+    clickHandler(e: MouseEvent): boolean | void;
+    // (undocumented)
+    connectedCallback(): void;
+    get displayValue(): string;
+    // (undocumented)
+    focusoutHandler(e: FocusEvent): boolean | void;
```

Comment:
should it be "focusOutHandler" and "keyDownHandler"? what's the pattern you're following?

Reply:
I'm following the string values for the events, which are "focusout" and "keydown".